api/src/service/domain/document/document_download.spec.ts

```diff
@@ -7,6 +7,7 @@ import { DocumentReference, StoredDocument, UploadedDocument } from "./document"
 import * as DocumentShared from "./document_shared";
 import { getDocument } from "./workflowitem_document_download";
 import VError from "verror";
+import { DocumentStorageServiceResponse } from "./document_upload";
```

Comment:
`DocumentStorageServiceResponse` is imported but never used

Reply:
Fixed